### DISCUSSION TITLE
On adding Google's ScaNN Index

### DISCUSSION THREAD

**Discussion by sgaseretto** [milvus-io/milvus#4706] [Ideas & Feature requests]
Are there any plans on adding support for [Google's ScaNN Index](https://ai.googleblog.com/2020/07/announcing-scann-efficient-vector.html) to Milvus? It seems to be the most performant ANN Index right now in terms of speed and accuracy.

**Comment by yhmo** [comment]
We have an issue about this: #2771 
We did some testing with ScaNN index, but the work is stopped because of short resource. Now we are focusing on new architect, cloud-native milvus. The integration of ScaNN index is still in backlog, but not in recently plan.
